run_fastsurfer.sh

```diff
@@ -965,24 +968,19 @@ then
     then
       mask_name_manedit=$(add_file_suffix "$mask_name" "manedit")
       if [[ -e "$mask_name_manedit" ]] ; then mask_name="$mask_name_manedit" ; fi
-      cmd=($python "${fastsurfercnndir}/segstats.py" --segfile "$asegdkt_segfile"
-           --segstatsfile "$asegdkt_statsfile" --normfile "$norm_name"
+      cmd=($python "${fastsurfercnndir}/segstats.py" --segfile "$asegdkt_segfile" --normfile "$norm_name"
+           --lut "$fastsurfercnndir/config/FreeSurferColorLUT.txt" --sd "${sd}" --sid "${subject}"
            --threads "$threads_seg" --empty --excludeid 0
-           --sd "${sd}" --sid "${subject}"
-           --ids 2 4 5 7 8 10 11 12 13 14 15 16 17 18 24 26 28 31 41 43 44 46 47
-                 49 50 51 52 53 54 58 60 63 77 251 252 253 254 255 1002 1003 1005
-                 1006 1007 1008 1009 1010 1011 1012 1013 1014 1015 1016 1017 1018
-                 1019 1020 1021 1022 1023 1024 1025 1026 1027 1028 1029 1030 1031
-                 1034 1035 2002 2003 2005 2006 2007 2008 2009 2010 2011 2012 2013
-                 2014 2015 2016 2017 2018 2019 2020 2021 2022 2023 2024 2025 2026
-                 2027 2028 2029 2030 2031 2034 2035
-           --lut "$fastsurfercnndir/config/FreeSurferColorLUT.txt"
-           measures --compute "Mask($mask_name)" "BrainSeg" "BrainSegNotVent"
-                              "SupraTentorial" "SupraTentorialNotVent"
-                              "SubCortGray" "rhCerebralWhiteMatter"
-                              "lhCerebralWhiteMatter" "CerebralWhiteMatter"
-           # make sure to read white matter hypointensities from the
-           )
+           --ids 2 4 5 7 8 10 11 12 13 14 15 16 17 18 24 26 28 31 41 43 44 46 47 49 50 51 52 53 54 58 60 63 77
+                 251 252 253 254 255
```

Comment:
These are CC ids, I thought they are not in the VINN segmentation and therefore we can omit them for now. or is $asegdkt_segfile later in the stream?

Reply:
Yes, this is the "aseg+DKT.stats" file. It is replicating the aseg format and has therefore had CC ids in there since the beginning.
I can obviously remove them, but why change the format of an existing and established file.